backend/remote/remote.go

```diff
@@ -176,6 +176,7 @@ func (r *Remote) GetVolUsage() (types.VolUsage, error) {
 	}
 
 	err = json.NewDecoder(resp.Body).Decode(&Details)
+	volUsage.RevisionCounter, _ = strconv.ParseInt(Details.RevisionCounter, 10, 64)
```

Comment:
shouldn't we handle the error from strconv ?

Reply:
@ashishranjan738 there are only two cases when it fails
1. If the length is > len(uint64/int64) or if the length is zero
for both the case we want the output to be zero, that's why not caching the error.